version.tf

```diff
@@ -6,8 +6,8 @@
 
 terraform {
   required_version = "~> 0.11.1"
-}
 
-provider "aws" {
-  version = "~> 1.60.0"
+  required_providers = {
```

Comment:
Should we mention this in the CHANGELOG or is it the equivalent to `provider` which we had before so isn't a visible change to users?

Reply:
this is equivalent to previous change so no change from user side.